doc/dune

```diff
@@ -1,2 +1,11 @@
 (documentation
  (package ppxlib))
+
+(rule
+ (alias doc)
+ (deps
+   (glob_files ./images/*))
+ (action
+   (progn
+     (system "mkdir -p %{project_root}/_doc/_html/ppxlib/assets/images")
+     (system "cp -R ./images/ %{project_root}/_doc/_html/ppxlib/assets/images/"))))
```

Comment:
Btw, I didn't know any other way to get assets on odoc. Do you guys happen to know?